index.html

```diff
@@ -84,6 +84,263 @@ <h1 class="primary-headline-text">Hey there. I’m Ansderly. <br>
         </div>
     </section>
 
+    <section class="recent-work-container">
+        <div class="recent-work-title-container">
+            <div class="recent-work-title">
+                <h1 class="text-recent-work">My Recent Works</h1>
+                <div class="line-work-container">
+                    <div class="line-work"></div>
+                </div>
+            </div>
+        </div>
+
+
+        <div class="single-card-container">
+            <div class="single-card">
+                <div class="img-single-work">
+                </div>
+                <div class="infos-container">
+                    <div class="name-single-work">
+                        <h1>Multi-Post Stories Gain+Glory</h1>
```

Comment:
- [Optional] Dear @hansderly, using more than one `h1` tag on a single page confuses a screen reader as to which one is the most important part. I prefer using `h2 - h6` right here for better semantic meaning.